tests/shutdown/shutdown.pm

```diff
@@ -36,6 +36,14 @@ sub test_flags {
     return {fatal => 1};
 }
 
+sub post_fail_hook {
+    my ($self) = @_;
+    # In case plymouth splash shows up and the shutdown is blocked, show
+    # console logs - save screen of console (plymouth splash screen in disabled at boottime)
+    send_key('esc');
+    save_screenshot;
```

Comment:
Shouldn't we add a call to parent's post fail hook to still try to collect logs, (like $self->SUPER::post_fail_hook) ?

Reply:
I've just copied over the existing post-fail-hook from x11/shutdown to have at least something. But since this can only fail in the middle of a shutdown I'd say our common post_fail_hook makes no sense here -> we don't have any ttys to switch to anymore ;)